queries/javascript/locals.scm

```diff
@@ -6,6 +6,10 @@
 (arrow_function) @scope
 (function_declaration) @scope
 (method_definition) @scope
+(if_statement) @scope
```

Comment:
This one is already covered by the `statement_block` in the consequence.

Reply:
I understood as follows.
- if_statement: Not required (because it is covered by statement_block)
- for_statement: add for_in_statement
- switch_statement: Not required (because the scope is not actually created)
- try_statement: changed to catch_clause (because try_statement doesn't actually create scope, but catch_clause does)

Is this perception correct?

Reply:
I fixed it as above
Please check again.